services/wallet/router/pathprocessor/processor_swap_paraswap.go

```diff
@@ -52,6 +52,8 @@ func getPartnerAddressAndFeePcnt(chainID uint64) (common.Address, float64) {
 		return common.HexToAddress("0x9a8278e856C0B191B9daa2d7DD1f7B28268E4DA2"), partnerFeePcnt
 	case walletCommon.BaseMainnet:
 		return common.Address{}, 0
+	case walletCommon.BSCMainnet:
+		return common.Address{}, 0
```

Comment:
We need Finance to generate a multisig on BSC to receive partner fees.
@Jas0n-F could you please take care of that?

Reply:
@smohamedjavid, we got the address for BSC, it's `0xEF693aCC26e7fb24B96056b33472D89d7dA5bAC9`

Reply:
Adding it 🙏